src/hub.html

```diff
@@ -1367,16 +1360,15 @@
                 ></a-entity>
             </a-entity>
         </a-entity>
-        <a-entity id="viewing-rig" set-yxz-order matrix-auto-update>
+        <a-entity id="viewing-rig" set-yxz-order>
```

Comment:
It looks like the dirty flag on this and the viewing-camera should be getting set by the Camera System, though the logic is a bit hard to follow so hard to say if there are edge cases this was capturing before.

Reply:
Yes, the dirty flag for the viewing-camera is set every frame  in `camera-system`.

https://github.com/mozilla/hubs/blob/856a2c36efb54685867063c698aa280ac40b99b4/src/systems/camera-system.js#L404-L406

We need to optimize it and don't set if the viewing-camera doesn't move tho.